File: test/scripts/perf/fib.lua
function fib(n)
  if n < 2 then return n end
  return fib(n - 2) + fib(n - 1)
end

for i = 1, 10 do
  local start = os.clock()
  io.write(string.format("fib: %d time: %.8f\n",fib(34), os.clock() - start))
end

File: test/language/benchmark/fib.py
import time

def fib(n):
	if n <= 2:
		return 1
	else:
		return fib(n - 1) + fib(n - 2)


for x in range(1,10):
    start_time = time.time()
    print("fib: %s time: %s s" % (fib(34),time.time() - start_time))
